File: C, C#, C++/C++/distribute-candies.cpp
class Solution:
    def distributeCandies(self, candyType: List[int]) -> int:
        set <int> candy (candyType.begin(), candyType.end())
        if candy.size()>= candyType.size()/2:
            return candyType.size()/2
        
        else:
            return candy.size()
        

File: Python/distribute-candies.py
class Solution:
    def distributeCandies(self, candyType: List[int]) -> int:
        candy = set (candyType)
        if len(candy)>= len(candyType)/2:
            return int (len(candyType)/2)
        
        else:
            return int (len(candy))
        

        
        
